code/modules/mapping/space_management/space_reservation.dm

```diff
@@ -52,6 +52,10 @@
 	for(var/turf/reserved_turf as anything in release_turfs)
 		SEND_SIGNAL(reserved_turf, COMSIG_TURF_RESERVATION_RELEASED, src)
 
+		// immediately disconnect from atmos
+		reserved_turf.blocks_air = TRUE
+		reserved_turf.immediate_calculate_adjacent_turfs()
```

Comment:
why the immediate? delaying seems healthy to me

Reply:
I'd rather be strict in the present and ease up over time; but I'll do it the other way for ya

Reply:
```suggestion
```

Reply:
use `CALCULATE_ADJACENT_TURFS(turf, KILLS_EXCITED)`

Reply:
why was this marked as resolved?